docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '2'
+
+services:
+  wordpress:
+    build: docker
```

Comment:
I think it is better to have Dockerfile in the root directory and then you can just use `build: ./`.